www/src/pages/foundations/spacing.tsx

```diff
@@ -140,7 +155,9 @@ export default function SpacingPage() {
                   step={0.5}
                   max={6}
                   value={size}
-                  onChange={e => setSize(e.target.value)}
+                  onChange={(e: {
+                    target: { value: React.SetStateAction<number>;
```

Comment:
`onChange={(e: React.ChangeEvent<HTMLInputElement>) => ...`

Reply:
Yes, thanks

Reply:
I think using `React.ChangeEvent<HTMLInputElement>` may have caused that TS2345 error because the `size` state variable expects a number per its definition. `e.target.value` is a string, so TypeScript is complaining that you're passing a string to a state variable that wants a number. If you used `parseInt(e.target.value)` in the `setSize` call with `React.ChangeEvent`, I wonder if that would resolve the error?

Reply:
Corrected